src/context/useSettings.tsx

```diff
@@ -1,12 +1,23 @@
 import { useQuery } from "@tanstack/react-query";
 import { queryKeys } from "util/queryKeys";
-import { fetchSettings } from "api/server";
-import type { LxdSettings } from "types/server";
+import { fetchSettingsFromClusterMembers, fetchSettings } from "api/server";
+import type { LXDSettingOnClusterMember, LxdSettings } from "types/server";
 import { UseQueryResult } from "@tanstack/react-query";
+import { LxdClusterMember } from "types/cluster";
 
 export const useSettings = (): UseQueryResult<LxdSettings> => {
   return useQuery({
     queryKey: [queryKeys.settings],
-    queryFn: fetchSettings,
+    queryFn: () => fetchSettings(),
+  });
+};
+
+export const useClusteredSettings = (
+  memberNames: LxdClusterMember[],
```

Comment:
```suggestion
```
I think this should not get any input, we can move the `useClusterMembers` call into this function to increase cohesion as we don't need the members in the caller.